tests/data/fga-mod-transformer-cases.yaml

```
@@ -172,3 +172,58 @@
       column:
         start: 4
         end: 12
+- name: contents path contains dot-dot-slash sequence and other invalid paths
+  modFile: |
+    schema: "1.2"
+    contents:
+      - core.fga
+      - ../team1/project.fga
+      - team2/../team3/board.fga
+      - team2\\..\\team3\\board.fga
+      - /absolute/path.fga
+      - %2e%2e%2fteam4%2fproject.fga
+      - %2e%2e%5cteam4%5cproject.fga
```

It looks like we're getting some yaml parse errors due to these starting with `%`, so we probably have to wrap them in quotes

```
* validation error at line=8, column=5: Plain value cannot start with directive indicator character % at line 8, column 5:
- %2e%2e%2fteam4%2fproject.fga
    ^
* validation error at line=9, column=5: Plain value cannot start with directive indicator character % at line 9, column 5:
- %2e%2e%5cteam4%5cproject.fga
    ^
```

```suggestion
      - "%2e%2e%2fteam4%2fproject.fga"
      - "%2e%2e%5cteam4%5cproject.fga"
```

---

Accepted the change, now will require an approval to run workflows